editor/src/core/shared/element-path-tree.ts

```diff
@@ -133,6 +134,73 @@ export function reorderTree(trees: ElementPathTrees, metadata: ElementInstanceMe
   }
 }
 
+function maybeReorderDynamicChildren(
+  children: ElementPathTree[],
+  metadata: ElementInstanceMetadataMap,
+): ElementPathTree[] {
+  function keepOnlyUnique(path: ElementPath, index: number, array: ElementPath[]) {
+    return array.indexOf(path) === index
```

Comment:
array.indexOf uses reference equality, shouldn't this use `find` and `EP.pathsEqual`?

Reply:
https://github.com/concrete-utopia/utopia/pull/3811/commits/96eb805a1f7aa28c783f6ba30d1e50a05b4bf9cc